src/Common/src/TypeSystem/Common/WellKnownType.cs

```diff
@@ -40,5 +40,6 @@ public enum WellKnownType
         RuntimeFieldHandle,
 
         Exception,
+        IndexOutOfRangeException,
```

Comment:
The definition of IndexOutOfRange exception definition should be local to the IL providers. The type system as whole should not need to know anything about the IndexOutOfRange exception.